astroid/brain/brain_argparse.py

```diff
@@ -29,6 +30,6 @@ def _looks_like_namespace(node):
     return False
 
 
-MANAGER.register_transform(
+AstroidManager().register_transform(
```

Comment:
I'm probably missing something, but isn't the idea to use the `MANAGER` singleton here?

Reply:
As it's a singleton/borg we can also instantiate.

Reply:
But isn't `MANGER` the singleton. `AstroidManger()` will create a new instance of `AstroidManager`.

Reply:
I think that the principle of the singleton is to get the same unique instance when you instantiate wherever you want, so you don't need to keep track of a global. I don't think this take much more time this way as ``self.__dict__`` will already be set. How do you use the singleton pattern ?

Reply:
> I don't think this take much more time this way as self.__dict__ will already be set.

I didn't saw that. You're right, that it will create a shared state which would work.

> How do you use the singleton pattern ?

I expected something like this
```py
class Manager:
    _instance = None

    def __new__(cls):
        if cls._instance is not None:
            cls._instance = super().__new__(cls)
        return cls._instance

Manager() == Manager()  # true
```

Reply:
Ha, yes it's a "borg" not a singleton, so the implementation is a little different. I had to check what a borg is in fact because I did not know. I don't think it was clear for other astroid's dev either because there was a MANAGER constants in various module. Originally I just wanted to change the import. Apparently a borg is better for multiprocessing (?)

Reply:
> Apparently a borg is better for multiprocessing (?)

Don't know about that. Had to look up what a `borg` is myself. We can keep it like this then